### PR TITLE
Tutorial 01: Click-by-click demonstration of worker area

### PR DESCRIPTION
**Pull request by pospec4444**
Tell player about work area as requested in #3127.

- Demonstrate work area switch during lumberjack's hut construction.

### CODE REVIEW

**Comment by gunchleoc** [comment]
The instructions before the new task tell the player to close the building window. This instruction should be moved to "Waiting for the lumberjack to go up".

**Comment by pospec4444** [comment]
Sorry to say that, but I don't understand the review (again) :-(
(I was reading it several times for like five minutes.)

What do you want me to move and where to?
Do you want the player to close the building window manually instead of `close_windows()`?

**Comment by gunchleoc** [comment]
When you place the tutorial, after placing the flag for the lumberjack, there is a message telling you to open the Lumberjack's window and to close it when you're done looking at it, followed immediately be the new message that you just added. So, telling the player to close the window doesn't make sense, because it needs to stay open for the demonstration of the working area.

**Comment by pospec4444** [comment]
Thanks for elaborating on the review. Let's hope that I got your idea :-)

Updated PR:
- Changed texts about closing construction site window
- Continue tutorial after player closes construction site window

________________________
If we want to put more accent on work areas, let's do it again in other tutorial. Naturally, it should be mentioned in Basic Controls because it **is** a basic control. On the other hand, work areas are crucial for some buildings (farms etc.) which are used later (Economy, Barbarians 01).

TL;DR: Shall we repeat work area in other tutorials?

File: data/campaigns/tutorial01_basic_control.wmf/scripting/texts.lua [hunk 1]
```diff
@@ -200,13 +200,13 @@ obj_lumberjack_progress = {
    title=_"Let’s see the progress",
    number = 1,
    body = objective_text(_"Let’s see the progress",
-      li(_[[Click on the construction site to have a look at it, then close its window again when you have seen enough.]]) ..
-      li_arrow(_[[To close the construction site’s window, simply right-click on it.]])
+      li(_[[Click on the construction site to have a look at it.]]) ..
+      li_arrow(_[[When you have seen enough, close the construction site’s window: simply right-click on it.]])
```
Comment (gunchleoc) [inline data/campaigns/tutorial01_basic_control.wmf/scripting/texts.lua]: This line should still be removed. We should not suggest to the player to close the window when it needs to stay open for the workarea demonstration.

File: data/campaigns/tutorial01_basic_control.wmf/scripting/texts.lua [hunk 2]
```diff
@@ -217,6 +217,19 @@ lumberjack_message_07 = {
    w = 350
 }
 
+lumberjack_message_07b = {
+   title = _"The Construction Site",
+   position = "topright",
+   body = (
+      li_image("images/wui/buildings/toggle_workarea.png",
+         _[[This button toggles building’s range. Buildings like lumberjack’s hut cover only some work area. Let me show you how perfect this spot is.]]) ..
```
Comment (gunchleoc) [inline data/campaigns/tutorial01_basic_control.wmf/scripting/texts.lua]:
```suggestion
         _[[This button toggles showing the building’s work area. When the lumberjack leaves the building to look for trees to fell, he will only look for them within this area. Let me show you how perfect this spot is.]]) ..
```

File: data/campaigns/tutorial01_basic_control.wmf/scripting/texts.lua [hunk 3]
```diff
@@ -217,6 +217,19 @@ lumberjack_message_07 = {
...
+      p(_[[Close construction site window when you have seen enough.]]) ..
```
Comment (gunchleoc) [inline data/campaigns/tutorial01_basic_control.wmf/scripting/texts.lua]:
```suggestion
      p(_[[Close the construction site window when you have seen enough.]]) ..
```

File: data/campaigns/tutorial01_basic_control.wmf/scripting/texts.lua (hunk 1 above)
Comment (pospec4444) [reply data/campaigns/tutorial01_basic_control.wmf/scripting/texts.lua]: It didn't seem inappropriate when I tested the scenario. Moreover, the objective is actually completed when player closes the construction site window.
On the other hand, nobody will probably get stuck there to seek help in Objectives window.

**Comment by pospec4444** [comment]
> `When you have seen enough, close the construction site’s window...`

Removed.

**Comment by klaus-halfmann** [comment]
Played the tutorial once again on this branch, fine for me. But I am not new to the game so I am not such a carefull tester.

**Review by Noordfrees — APPROVED** [review]
Played the tutorial with this branch, seems fine to me. I'll merge this now…
